anesthetic/plot.py

```diff
@@ -268,13 +268,15 @@ def fastkde_plot_1d(ax, data, *args, **kwargs):
 
     xmin = kwargs.pop('xmin', None)
     xmax = kwargs.pop('xmax', None)
+    q = kwargs.pop('q', '5sigma')
+    q = quantile_plot_interval(q=q)
 
     try:
         x, p = fastkde_1d(data, xmin, xmax)
     except NameError:
         raise ImportError("You need to install fastkde to use fastkde")
     p /= p.max()
-    i = (x < quantile(x, 0.99, p)) & (x > quantile(x, 0.01, p)) | (p > 0.1)
+    i = ((x > quantile(x, q[0], p)) & (x < quantile(x, q[1], p)))
```

Comment:
You removed the `| (p>0.)` code here. I think the intention of this was to make sure that if the posterior was pressed up against a boundary you didn't clip it nearby. Was there a reason you removed it?

Reply:
This gave unexpected output when specifying `q`, e.g. even though specifying `q='sigma1'` it would plot up to almost `'sigma3'`.

Reply:
That makes sense.

Reply:
Do you think the `p>0.1` is really needed?

Reply:
No. Keep an eye on it as you use it, but I think it was an attempt to address the issues which the rest of this PR does